apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/grpc/GrpcMetadataResolver.java

```diff
@@ -19,15 +19,35 @@ public class GrpcMetadataResolver {
   /** Metadata key for passing the cassandra token to the Bridge. */
   private final Metadata.Key<String> cassandraTokenKey;
 
+  /** Default metadata. */
+  private final Metadata defaultMetadata;
+
+  // TODO add validation @Pattern(regexp = "rest|graphql")  after
```

Comment:
We probably either want to just validate legal characters, or make set of acceptable ids configurable, to allow addition of new APIs by other developers?
If it was configurable we could add our own regex for production deployment to ensure no unknown ids were to be used.

Reply:
Does not make sense to be configurable, because it should no be allowed that somebody changes the source API for the REST to `graphql`.. Adding a new value is another issue and quite complicated at the moment..

Reply:
Yeah, sorry, should have removed comment after I realized complexity of enum mapping etc.
You are absolutely right.
There is the bigger question on extensibility, but this PR is not the place for that.